install_config/storage_examples/shared_storage.adoc

```diff
@@ -17,21 +17,21 @@ highlights the use of NFS, but can easily be adapted to other shared storage
 types, such as GlusterFS. In addition, this example will show configuration of
 pod security as it relates to shared storage.
 
-For an explanation of persistent volumes (PVs), persistent volume claims (PVCs),
-and using NFS as persistent storage, see the section on
-link:../persistent_storage/persistent_storage_nfs[Persistent Storage Using NFS].
-This topic shows and end-to-end example of using an existing NFS cluster and
+link:../../install_config/persistent_storage/persistent_storage_nfs.html[Persistent
+Storage Using NFS] provides an explanation of persistent volumes (PVs),
+persistent volume claims (PVCs), and using NFS as persistent storage. This topic
+shows and end-to-end example of using an existing NFS cluster and
 {product-title} persistent store, and assumes an existing NFS server and exports
 exist in your {product-title} infrastructure.
```

Comment:
Maybe we should clarify in the Overview that these steps are really aimed at a cluster administrator, because only cluster-admins can create PVs in the cluster (though any user could created PVCs).